framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -205,6 +225,32 @@ public void updateForMethodCall(
             MethodInvocationNode n, AnnotatedTypeFactory atypeFactory, V val) {
         ExecutableElement method = n.getTarget().getMethod();
 
+        // List of expressions that this method side-effects (specified as arguments/elements of
+        // @SideEffectsOnly). If the list is empty, then there is no @SideEffectsOnly annotation.
+        List<JavaExpression> sideEffectsOnlyExpressions = new ArrayList<>();
+        if (isSideEffectsOnly(atypeFactory, method)) {
+            SourceChecker checker = analysis.checker;
+            JavaExpressionParseUtil.JavaExpressionContext methodUseContext =
+                    JavaExpressionParseUtil.JavaExpressionContext.buildContextForMethodUse(
+                            n, checker);
+
+            AnnotationMirror sefOnlyAnnotation =
```

Comment:
What does the `f` stand for?  Is it just copied from code about **S**ide**E**ffect**F**ree?  You might as well name this variable `sideEffectsOnlyAnnotation` which is clearer.